src/osx/core/bitmap.cpp

```diff
@@ -1543,23 +1543,23 @@ void wxMask::Init()
 
 void *wxMask::GetRawAccess() const
 {
-    wxCHECK_MSG( m_maskBitmap, nullptr , wxT("invalid mask") ) ;
+    wxCHECK_MSG( m_maskBitmap, nullptr , "invalid mask" ) ;
     return CGBitmapContextGetData(m_maskBitmap);
 }
 
 int wxMask::GetBytesPerRow() const
```

Comment:
This one should be changed to return `size_t`, it ends up being passed as `size_t` argument to e.g. `CGBitmapContextCreate()`, so it doesn't make much sense to cast to `int` when we need `size_t` anyhow.

Reply:
I'm OK with just removing the "Add casts to suppress implicit conversion warnings" commit. `GetBytesPerRow()` is used in a lot of places where it is assigned to `int` and I would rather not change the API and risk even more warnings.

Reply:
I reverted the int/size_t stuff. The other changes are innocuous.